zerver/tests/test_signup.py

```diff
@@ -1202,14 +1203,14 @@ def test_invitation_reminder_email(self) -> None:
         self.assertEqual(len(email_jobs_to_deliver), 1)
         email_count = len(outbox)
         for job in email_jobs_to_deliver:
-            send_email(**ujson.loads(job.data))
+            deliver_email(job)
         self.assertEqual(len(outbox), email_count + 1)
         self.assertIn(FromAddress.NOREPLY, outbox[-1].from_email)
 
         # Now verify that signing up clears invite_reminder emails
         email_jobs_to_deliver = ScheduledEmail.objects.filter(
             scheduled_timestamp__lte=timezone_now(), type=ScheduledEmail.INVITATION_REMINDER)
-        self.assertEqual(len(email_jobs_to_deliver), 1)
+        self.assertEqual(len(email_jobs_to_deliver), 0)
```

Comment:
I think this change makes the test no do what was intended -- we're no longer checking that the invitation_reminder was cleared?